src/sentry/tempest/models.py

```diff
@@ -26,7 +26,7 @@ class TempestCredentials(DefaultFieldsModel):
     # used to communicate the status of the latest actions with credentials
     message = models.TextField()
     message_type = models.CharField(
-        max_length=20, choices=MessageType.choices, default=MessageType.ERROR
+        max_length=20, choices=MessageType.choices, default=None, null=True
```

Comment:
Could you maybe explain in a comment why `null` exists instead of a state representing it?

Reply:
The idea is that in the beginning it is null in which case the UI displays a tag saying "pending" once it fails the message becomes type error in which case  the UI displays a tag saying "error" and once it works it becomes message type success and a success is displaying.